dist/page-notice/page-notice.css

```diff
@@ -10,10 +10,11 @@
 }
 div[role="region"].page-notice,
 section.page-notice {
-  display: flex;
+  display: grid;
```

Comment:
The alignment was not the reason to switch to grid. The main issue was the `cta` element changing location based on viewport width breakpoints. I tried various kinds of attempts doing that with `flex` and none panned out, at least not without some significant undesired hacking which would include lifting the close icon off the page (absolute), moving elements around in the DOM (would create an unnatural tab order - a11y issue), etc. The `cta` comes before the `X` and on smaller screens it needs to slip under the message leaving the `X` in the same place. Creating a break isn't hard. Creating a break for one element and leaving the following element to follow the previous flow/row I could not figure out.

In the end, `flex` seemed too linear. I think `grid` seems like the better option for this sort of thing overall.

That said, if you can see an easy way to solve this with `flex` that I've perhaps overlooked, I'm not opposed to going back to `flex`.

Reply:
I spoke to Arthur about this as well. I agree with the grid solution. The issue is that the container between the dismiss button and the text (the link part) needs to shift down. It might be possible with flex, but I think grid is the cleanest solution.